pydemo/test.py

```diff
@@ -0,0 +1,42 @@
+import signal
+import sys
+import time
+import numpy as np
+import imu_core.imu_core_cpp as IMU
+
+
+def handle_signal(s, f):
+    # this way is better then the commented version because you want
+    # to force it to call destructor immediately
```

Comment:
```
    # this way is better then the commented version because you want
    # to force it to call destructor immediately
```

Should this comment be removed?